lib/commits.js

```diff
@@ -12,7 +12,7 @@ module.exports = tags =>
     const [release, parent] = tags
 
     if (!release || !parent || !parent.hash || !release.hash) {
-      reject(new Error('the first release should be created manually.'))
+      reject(new Error('the first two releases should be created manually.'))
```

Comment:
What? Why? It only checks if the tag for the current one exists and if the first release exists on GitHub Releases. Only the first release needs to exist there... 😊